The subject error occurs because in the new c++11 a space between string concatenation is mandatory
Index: oyranos-0.4.0/examples/xcmsevents/qcmsevents.cpp
===================================================================
--- oyranos-0.4.0.orig/examples/xcmsevents/qcmsevents.cpp	2011-11-10 16:42:12.000000000 +0100
+++ oyranos-0.4.0/examples/xcmsevents/qcmsevents.cpp	2012-10-10 21:47:57.017451573 +0200
@@ -118,8 +118,8 @@
   else if (XCME_MSG_DISPLAY_STATUS)
   {
     int i;
-    if(strstr(text, "atom: \""XCM_COLOR_DESKTOP"\":"))
-      sscanf( text, "atom: \""XCM_COLOR_DESKTOP"\": %d %d", &i, &pid );
+    if(strstr(text, "atom: \"" XCM_COLOR_DESKTOP "\":"))
+      sscanf( text, "atom: \"" XCM_COLOR_DESKTOP "\": %d %d", &i, &pid );
     if(pid == 0)
     {
       colour.setHsvF( 0.6, 0.4, 0.9 );
